src/components/button/createButton.ts

```diff
@@ -2,33 +2,64 @@ import { ComposeFactory } from 'dojo-compose/compose';
 import { VNodeProperties } from 'dojo-interfaces/vdom';
 import createWidgetBase from '../../createWidgetBase';
 import { Widget, WidgetOptions, WidgetProperties, WidgetState } from './../../interfaces';
-import createFormFieldMixin, { FormFieldMixin, FormFieldMixinState, FormFieldMixinOptions } from '../../mixins/createFormFieldMixin';
 
-export interface ButtonState extends WidgetState, FormFieldMixinState<string> {
-	label?: string;
+export interface ButtonState extends WidgetState {
+	/*
+	 * Text content of button
+	 */
+	content?: string;
+
+	/**
+	 * The button's name
+	 */
+	name?: string;
+
+	/**
+	 * Accessibility attributes
+	 */
+	disabled?: boolean;
+	pressed?: boolean;
 }
 
 export interface ButtonProperties extends WidgetProperties {
-	label?: string;
+	content?: string;
+	name?: string;
+	type?: string;
+
+	/**
+	 * Accessibility attributes
+	 */
+	descriptionID?: string;
+	disabled?: boolean;
```

Comment:
I would argue that disabled is not an accessibility attribute, but of course would be used to add an accessibility attribute plus other things?